airbyte-integrations/connectors/source-sparkpost/manifest.yaml

```diff
@@ -0,0 +1,1007 @@
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestPath
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: "{{ response.get('next', {}).get('cursor') }}"
+            stop_condition: "{{ response.get('results', []) | length == 0 }}"
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/rejection_reason_metrics"
+    metrics:
+      type: DeclarativeStream
+      name: metrics
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /api/v1/metrics/
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path: []
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/metrics"
+    metrics_by_template:
+      type: DeclarativeStream
+      name: metrics_by_template
+      primary_key:
+        - template_id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /api/v1/metrics/deliverability/template
+          http_method: GET
+          request_parameters:
+            from: "{{ config[\"from\"] }}"
+            metrics: "{{ config[\"metrics\"] }}"
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - results
+        paginator:
+          type: DefaultPaginator
+          page_token_option:
+            type: RequestPath
+          pagination_strategy:
+            type: CursorPagination
+            cursor_value: >-
+              {{ response.get('links', []) | selectattr('rel', 'equalto',
+              'next') | map(attribute='href') | first }}
+            stop_condition: >-
+              {{ not response.get('links', []) | selectattr('rel', 'equalto',
+              'next') | list }}
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/metrics_by_template"
+    snippets:
+      type: DeclarativeStream
+      name: snippets
+      primary_key:
+        - id
+      retriever:
+        type: SimpleRetriever
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: /api/labs/snippets
+          http_method: GET
+        record_selector:
+          type: RecordSelector
+          extractor:
+            type: DpathExtractor
+            field_path:
+              - results
+      schema_loader:
+        type: InlineSchemaLoader
+        schema:
+          $ref: "#/schemas/snippets"
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.sparkpost.com
```

Comment:
Sparkpost have two endpoints one for global and other for EU region, check [here](https://developers.sparkpost.com/api/#header-endpoints)

Reply:
@marcosmarxm they both work independently with diff api keys. If one signedup for global they has to use it.

Reply:
What I'm saying your connector must have a `region` parameter. You don't need to test EU but prepare it to work.